### PR TITLE
ArUco 1.3.0 (new formula)

### PR DESCRIPTION
**Pull request by pierre-rouanet**
Add support for the ArUco library : a minimal library for Augmented Reality applications based on OpenCV
http://www.uco.es/investiga/grupos/ava/node/26

### CODE REVIEW

**Comment by tseemann** [comment]
@BrewTestBot test this please

File: aruco.rb [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  homepage "http://www.uco.es/investiga/grupos/ava/node/26"
+  url "https://downloads.sourceforge.net/projects/aruco/files/1.3.0/aruco-1.3.0.tgz"
+  sha256 "a71d2138bfe77bf85c0b162979fe87e342e3133bef93ac319079ecbafbd329ed"
+
+  depends_on "cmake" => :build
+  depends_on "opencv"
+
+  def install
+    system "cmake", ".", *std_cmake_args
+    system "make", "install"
```
Comment (dpo) [inline aruco.rb]: Is it possible to run `make check` or `make test` before `make install`?

**Comment by pierre-rouanet** [comment]
I did not find any way to add some test or check before the install.

**Comment by dpo** [comment]
@BrewTestBot test this please

**Comment by dpo** [comment]
```
==> Verifying aruco-1.3.0.tgz checksum
Error: SHA256 mismatch
Expected: a71d2138bfe77bf85c0b162979fe87e342e3133bef93ac319079ecbafbd329ed
Actual: 1ff21334e7ec143dfbfe39a458511d21c4c9d363fb176a09739a8d657e0f3cec
Archive: /Library/Caches/Homebrew/aruco-1.3.0.tgz
To retry an incomplete download, remove the file above.
==> brew audit Homebrew/science/aruco --strict --online			 FAILED
==> brew style aruco

1 file inspected, no offenses detected
Error: 2 problems in 1 formula

==> audit problems
homebrew/science/aruco:
 * Stable: Please use https:// for http://downloads.sourceforge.net/project/aruco/1.3.0/aruco-1.3.0.tgz
 * Description is too long. "name: desc" should be less than 80 characters.
```

**Comment by dpo** [comment]
@pierre-rouanet

**Comment by dpo** [comment]
Closing in favor of #4456.

**Pull request closed without merging**